src/avro_ocf.erl

```diff
@@ -146,12 +145,15 @@ make_header(Type) ->
 -spec make_header(avro_type(), extra_meta()) -> header().
 make_header(Type, ExtraMeta0) ->
   ExtraMeta = validate_extra_meta(ExtraMeta0),
```

Comment:
The name `ExtraMeta` now is a bit confusing.
Would like to have it renamed to just 'Meta', and spec renamed to `meta()`.

Also, please update the docs for API functions (`make_header/2` and `write_file/5`)
to have compression option explained